### PR TITLE
RIA-7661 Appeal transferred to the First-tier Tribunal - HO notification

### PR DESCRIPTION
**Pull request by javierhmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RIA-7661

### Change description ###

Added code functionality and test scenarios

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated/added (if needed)
- [x] tests have been updated / new tests have been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by colmlynch60 — DISMISSED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/respondent/RespondentTurnOnNotificationsPersonalisation.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    private final String homeOfficeTurnOnNotificationsAfterListingTemplateId;
+    private final String iaExUiFrontendUrl;
+    private final PersonalisationProvider personalisationProvider;
+    private final CustomerServicesProvider customerServicesProvider;
+    private final HomeOfficeEmailFinder hoEmailAddressFinder;
+
+
+
+    public RespondentTurnOnNotificationsPersonalisation(
+            @NotNull(message = "endAppealHomeOfficeTemplateId cannot be null")
```
Comment (ThomasKKC) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/respondent/RespondentTurnOnNotificationsPersonalisation.java:36]: Can you rename these "endAppeal *" values

**Review by colmlynch60 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/respondent/RespondentTurnOnNotificationsPersonalisation.java (hunk 1 above)
Comment (colmlynch60) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/respondent/RespondentTurnOnNotificationsPersonalisation.java]: Change made

**Review by ThomasKKC — APPROVED** [review]

**Pull request merged**